src/module.ts

```diff
@@ -180,3 +186,15 @@ function pick<O extends Record<any, any>, K extends keyof O>(obj: O, keys: K[]):
   }
   return newobj
 }
+
+function generateImageOptions(providers: ImageModuleProvider[], imageOptions: Omit<CreateImageOptions, 'providers' | 'nuxt' | 'runtimeConfig'>): string {
+  return `
+  ${providers.map(p => `import * as ${p.importName} from '${p.runtime}'`).join('\n')}
+  
+  export const imageOptions = ${JSON.stringify(imageOptions, null, 2)}
+  
+  imageOptions.providers = {
```

Comment:
I _think_ this causes tree-shaker to not work properly with assignments (so code will be in bundle even if util not used but again just guess).